docs/Using-Fleet/Mobile-device-management.md

```diff
@@ -1,5 +1,16 @@
 # Mobile device management (MDM)
 
+- [Controls](#controls)
+  - [macOS updates](#macos-updates)
+- [Apple Push Notification Service](#apple-push-notification-service-apns)
+  - [APNs renewal](#apns-renewal)
+- [Apple Business Manager](#apple-business-manager-abm)
+  - [ABM renewal](#abm-renewal)
+- [Disk encryption key](#disk-encryption-key)
+  - [Viewing a disk encryption key](#viewing-a-disk-encryption-key)
+  - [Recovering a device using the disk encryption key](#recover-a-device-using-the-disk-encryption-key)
+
+
```

Comment:
```suggestion
```

Hey! Thanks for adding these (heads up play) but I don't think we need these anchor links. This is because the website automatically displays the the h2s (##) in the sidebar. 

Here's a screenshot of the current sidebar in the [docs](https://fleetdm.com/docs/using-fleet/mobile-device-management):
![Screenshot 2023-02-03 at 10 14 03 AM](https://user-images.githubusercontent.com/47070608/216638771-efa29989-2c67-4920-9dcd-80069d8c6209.png)